packages/office-ui-fabric-react/src/components/ComponentExamples.test.tsx

```diff
@@ -100,10 +102,15 @@ describe('Component Examples', () => {
   const realToLocaleString = global.Date.prototype.toLocaleString;
   const realToLocaleTimeString = global.Date.prototype.toLocaleTimeString;
   const realToLocaleDateString = global.Date.prototype.toLocaleDateString;
-  const constantDate = new Date(Date.UTC(2017, 13, 6, 4, 41, 20));
+  const constantDate = new Date(Date.UTC(2017, 0, 6, 4, 41, 20));
   const files: string[] = glob.sync(path.resolve(process.cwd(), 'src/components/**/examples/*Example*.tsx'));
 
   beforeAll(() => {
+    // Mock createPortal to capture its component hierarchy in snapshot output.
+    ReactDOM.createPortal = jest.fn(element => {
```

Comment:
pretty sure you're the only one testing with portals, so not an immediate issue here. But you should restore the implementation when you're all done inside an `afterAll()` block.

Reply:
There is a `jest.restoreAllMocks()` call in the `afterAll()` block. But that doesn't work as of the latest enzyme adapter so I need to restore a different way.